autokeras/hypermodel/resnet.py

```diff
@@ -0,0 +1,330 @@
+import tensorflow as tf
+
+from autokeras.hypermodel import hyper_block
+
+backend = tf.keras.backend
+layers = tf.keras.layers
+models = tf.keras.models
+utils = tf.keras.utils
```

Comment:
It is probably better by using `import tf.keras.backend as backend`

Reply:
Yes